indico/MaKaC/webinterface/rh/conferenceDisplay.py

```diff
@@ -101,26 +101,21 @@ def _process( self ):
         view = "static"
         wf = self.getWebFactory()
         if wf != None:
-            type = self.getWebFactory().getId()
+            evt_type = self.getWebFactory().getId()
```

Comment:
this seems the same logic used in `Conference.getType()` - any idea why this code doesn't simply call `self._conf.getType()`?